instrumentation/aws-lambda/aws-lambda-events-2.2/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/awslambdaevents/v2_2/AwsLambdaRequestHandlerInstrumentation.java

```diff
@@ -63,18 +61,24 @@ public static void onEnter(
         @Advice.Local("otelMessageContext") io.opentelemetry.context.Context messageContext,
         @Advice.Local("otelMessageScope") Scope messageScope) {
```

Comment:
This class is same as current instrumentation in HEAD

It's probably not enough to figure out a way to share logic in the advice classes between the two javaagent instrumentations at this time